src/include/network/lan/cmdline.rb

```diff
@@ -194,9 +194,9 @@ def AddHandler(options)
         [LanItems.current, "ifcfg"],
         Ops.get(options, "name", "")
       )
-      LanItems.type = NetworkInterfaces.device_type(
-        Ops.get(options, "name", "")
-      )
+      LanItems.type = options["type"]
+      raise "Device type is mandatory." if !LanItems.type
```

Comment:
this looks like API change to me. So now in CLI type is mandatory? If so, then please mention it in changes file

Reply:
yes. Bcs before it was "detected" from device name which is not possible anymore. In UI it is done by explicitly selecting by user ... so I had to introduce something similar here too.

Will add change log entry.